assets/src/edit-story/components/canvas/layout.js

```diff
@@ -141,7 +138,7 @@ const PageAreaDangerZone = styled.div`
   );
   opacity: 0.05;
   width: 100%;
-  height: calc((var(--fullbleed-height-px) - var(--page-height-px)) / 2);
+  height: var(--danger-zone-height-px);
```

Comment:
Also, this is expressible in `%`, right? I just did something like this in https://github.com/google/web-stories-wp/pull/2265

Reply:
Though, of course, if you need the `--danger-zone-height` anyway, we should just use it.